### PR TITLE
Initial add of files

### PR DESCRIPTION
**Pull request by vphill**
This is the first set of files.  I felt like I wanted to put this here.

### CODE REVIEW

**Comment by vphill** [comment]
Here is the structure I'm proposing for pyuntl

```
├── pyuntl
│   ├── NACO.py
│   ├── __init__.py
│   ├── dc_structure.py
│   ├── etd_ms_structure.py
│   ├── form_logic.py
│   ├── highwire_structure.py
│   ├── metadata_generator.py
│   ├── quality.py
│   ├── untl_structure.py
│   ├── untldoc.py
│   ├── util.py
│   └── validator.py
├── requirements.txt
├── runtests.py
├── setup.py
└── tests
    ├── __init__.py
    ├── dublincore.py
    ├── field.py
    ├── metadc_blank_description.untl.xml
    ├── metadc_complete.untl.xml
    ├── metadc_empty.untl.xml
    ├── metadc_legacy_defaults.untl.xml
    ├── metadc_no_description.untl.xml
    ├── pyuntl_test.py
    ├── reader.py
    ├── record.py
    └── writer.py
```

**Comment by vphill** [comment]
Make sure we add a license

**Comment by somexpert** [comment]
MANIFEST.in?

**Comment by ldko** [comment]
Do we need NACO.py?

**Comment by vphill** [comment]
@ldko we should be able to use pynaco instead going forward

**Comment by vphill** [comment]
ok @somexpert @ldko @damonkelley 

I think I've finished up the parts of this pull request that I know I needed to do.  

I'm sure there are other things we could do so next steps are welcome.

**Comment by somexpert** [comment]
Do you want to add a MANIFEST to this? Also, you might want to add versions to the requirements.

**Comment by somexpert** [comment]
Also, I believe normally, if you have data used by the tests, we tend to put them into a subdir in tests. Not sure it needs to be done here though.

File: README.md [hunk 1]
```diff
@@ -1,2 +1,27 @@
...
+License
+-------
+
+See LICENSE.txt
+
+
+Acknowledgements
+----------------
+
+The pyuntl was developed at the UNT Libraries and has been worked on by a number of developers over the years including:
```
Comment (somexpert) [inline README.md]: Don't think we need the initial 'The'.

**Comment by damonkelley** [comment]
looks good to me. just had 2 things to mention.

1. We probably want to add `dist/` and `*.egg-info` to `.gitignore`
2. `pyuntl/dc_structure.py` is an executable, but I don't think it needs to be.

**Comment by damonkelley** [comment]
>Also, I believe normally, if you have data used by the tests, we tend to put them into a subdir in tests. Not sure it needs to be done here though.

While this is true, it might not be an appropriate time for us to move them. The major difference between this package and many of our others is that this package already has a test suite, so moving those files will require changing the paths through out the suite.

**Comment by damonkelley** [comment]
>Do you want to add a MANIFEST to this? Also, you might want to add versions to the requirements.
Not sure if we need one right now. From inspecting the `pyunt.egg-info/SOURCES.txt` it looks like everything we need included or excluded is already taken care of.

**Comment by vphill** [comment]
Alright,  I think I've satisfied the comments and suggestions from above.  @ldko @somexpert @damonkelley  what say you?

**Comment by damonkelley** [comment]
LGTM :+1:

**Comment by somexpert** [comment]
:thumbsup:

**Pull request merged**